Library/Formula/gws.rb

```diff
@@ -0,0 +1,22 @@
+class Gws < Formula
+  homepage "https://streakycobra.github.io/gws/"
```

Comment:
Sorry, we've recently added a `desc` (description) field. Would you mind adding one? After that this should be good to go.